src/MaciVotingSetup.sol

```diff
@@ -112,37 +118,12 @@ contract MaciVotingSetup is PluginSetup {
             permissionId: DAO(payable(_dao)).EXECUTE_PERMISSION_ID()
         });
 
-        // TODO: Test which of these permissions are needed
         preparedSetupData.permissions[1] = PermissionLib.MultiTargetPermission({
```

Comment:
We could run it with just one permission (delete the `permission[1]`)

Reply:
deleted!